recipes/pymove/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "pymove" %}
+{% set version = "1.1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b6d8dfbe9a6b7f0cb8587a70db5bfc385c0a9395a5e430a61f38ec45198e5b77
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
I believe this package can be made `noarch: python`. See:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

In that case, change `python` to `python >=3.5` under `host` and `run`